storybookjs/storybook
# UI: Fix toggle button for custom theming

Issue: [UI: Fix toggle button for custom theming #13924](https://github.com/storybookjs/storybook/issues/13924)

**Fixes Boolean control change, Search input theme configuration, and arrows in the left nav.**

## What I did
Forked the storybook repo, and setup locally as specified [here](https://storybook.js.org/docs/react/contribute/code). 
When I was going through the code, I noticed that Control was taking the background parameters from **barBg**, the change will take the parameters from **app**.

## How to test

Run ```yarn test``` in storybook and all test suites and tests should pass.
![image](https://user-images.githubusercontent.com/56908732/124089658-b81d0000-da71-11eb-81c3-a620314cee9e.png)


- Is this testable with Jest or Chromatic screenshots?
  I am not sure, as I am new and contributing first time to storybook. 😅 
- Does this need a new example in the kitchen sink apps?
  No, I don't think it requires. But, I am not sure.
- Does this need an update to the documentation?
  No, according to me, it doesn't require an update to the documentation.
